purefa/README.md

```diff
@@ -0,0 +1,123 @@
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+       - host:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/volumes?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/hosts?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+
+  - openmetrics_endpoint: http://<exporter_ip_or_fqdn>:9491/metrics/flasharray/pods?endpoint=<array_ip_or_fqdn>
+    tags:
+       - env:<env>
+       - fa_array_name:<full_fqdn>
+       - host:<full_fqdn>
+    hostname_label: host
+    hostname_format: <HOSTNAME>.<domain_name>
+    headers:
+       Authorization: Bearer <api_token>
+    min_collection_interval: 120
+    empty_default_hostname: true
+    max_returned_metrics: 100000
+```
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `purefa` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this check.
+
+### Events
+
+The PureFA integration does not include any events.
+
+### Service Checks
+
+The PureFA integration does not include any service checks.
+
+## Support
+
+For support or feature requests, contact Pure Storage through the following methods:
+* Email: pure-observability@purestorage.com
+* Slack: [Pure Storage Code// Observability Channel][11].
+
+[1]: https://github.com/PureStorage-OpenConnect/pure-exporter
+[2]: https://app.datadoghq.com/account/settings#agent
+[4]: https://github.com/PureStorage-OpenConnect/observability/blob/master/datadog/integrations-extras/purefa/datadog_checks/purefa/data/conf.yaml.example
+[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[6]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
+[7]: https://github.com/PureStorage-OpenConnect/observability/blob/master/datadog/integrations-extras/purefa/metadata.csv
+[9]: https://app.datadoghq.com/account/settings#agent
+[10]: https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent
+[11]: https://code-purestorage.slack.com/messages/C0357KLR1EU
```

Comment:
Are new users able to sign up for your slack? This just brought me to a login page

Reply:
Yes - anyone can sign up for this Slack.